pynos/versions/base/interface.py

```diff
@@ -2926,3 +2925,251 @@ def get_port_chann_detail_request(last_aggregator_id):
+                method.  The only parameter passed to `callback` will be the
+                ``ElementTree`` `config`.
+
+        Returns:
+            Return value of `callback`.
+
+        Raises:
+            KeyError: if `int_type`, `name`, `vrid`, or `vip` is not passed.
+            ValueError: if `int_type`, `name`, `vrid`, or `vip` is invalid.
+
+        Examples:
+            >>> import pynos.device
+            >>> switches = ['10.24.39.211', '10.24.39.203']
+            >>> auth = ('admin', 'password')
+            >>> for switch in switches:
+            ...     conn = (switch, '22')
+            ...     with pynos.device.Device(conn=conn, auth=auth) as dev:
+            ...         output = dev.interface.anycast_mac(rbridge_id='225',
+            ...         mac='aabb.ccdd.eeff', delete=True)
+            ...         output = dev.services.vrrpe(ip_version='6',
+            ...         enabled=True, rbridge_id='225')
+            ...         output = dev.services.vrrpe(enabled=True,
+            ...         rbridge_id='225')
+            ...         output = dev.interface.set_ip('tengigabitethernet',
+            ...         '225/0/18', '10.1.1.2/24')
+            ...         output = dev.interface.ip_address(name='225/0/18',
+            ...         int_type='tengigabitethernet',
+            ...         ip_addr='2001:4818:f000:1ab:cafe:beef:1000:2/64')
+            ...         dev.interface.vrrpe_vip(int_type='tengigabitethernet',
+            ...         name='225/0/18', vrid='1', vip='10.1.1.1/24')
+            ...         dev.interface.vrrpe_vip(int_type='tengigabitethernet',
+            ...         name='225/0/18', vrid='1',
+            ...         vip='fe80::cafe:beef:1000:1/64')
+            ...         dev.interface.vrrpe_vip(int_type='tengigabitethernet',
+            ...         name='225/0/18', vrid='1',
+            ...         vip='2001:4818:f000:1ab:cafe:beef:1000:1/64')
+            ...         output = dev.interface.add_vlan_int('89')
+            ...         output = dev.interface.ip_address(name='89',
+            ...         int_type='ve', ip_addr='172.16.1.1/24',
+            ...         rbridge_id='225')
+           ...         output = dev.interface.ip_address(name='89',
+            ...         int_type='ve', rbridge_id='225',
+            ...         ip_addr='2002:4818:f000:1ab:cafe:beef:1000:2/64')
+            ...         dev.interface.vrrpe_vip(int_type='ve', name='89',
+            ...         vrid='1', vip='172.16.1.2/24', rbridge_id='225')
+            ...         dev.interface.vrrpe_vip(int_type='ve', name='89',
+            ...         vrid='1', vip='fe80::dafe:beef:1000:1/64',
+            ...         rbridge_id='225')
+            ...         dev.interface.vrrpe_vip(int_type='ve', name='89',
+            ...         vrid='1', vip='2002:4818:f000:1ab:cafe:beef:1000:1/64',
+            ...         rbridge_id='225')
+            ...         output = dev.services.vrrpe(ip_version='6',
```

Comment:
You pass in delete=True/False but I dont see support for it in the code? Has this doctest been tested?